packages/vite-plugin-react-router/package.json

```diff
@@ -16,6 +16,14 @@
         "types": "./dist/index.d.mts",
         "default": "./dist/index.mjs"
       }
+    },
+    "./function-handler": {
+      "types": "./dist/function-handler.d.mts",
+      "default": "./dist/function-handler.mjs"
+    },
+    "./edge-function-handler": {
```

Comment:
Check out the first lines of `FUNCTION_HANDLER` and `EDGE_FUNCTION_HANDLER` to understand what these are for. They're the _runtime_ entry points.

(This should probably always have been an explicit separate export, honestly.)